config/default.yml

```diff
@@ -3219,9 +3219,9 @@ Style/ExplicitBlockArgument:
                   that just passes its arguments to another block.
   StyleGuide: '#block-argument'
   Enabled: true
-  # May change the yielding arity.
-  Safe: false
+  Safe: true
```

Comment:
Safe is true by default, so it can be omitted.

```suggestion
```

Reply:
🤦 Fixed now, thanks.